pages/implementation/start-custom-destination.mdx

```diff
@@ -255,6 +324,34 @@ Call the initAvo function:
 Avo.initAvo({ env: Avo.AvoEnv.Dev }, {}, customDestination);
 ```
 
+### <a name="ReasonML"></a> ReasonML
+
+```ReasonML
+let customDestination: Avo.avoCustomDestination = {
+  make: (_env: Avo.avoEnvType) => (),
+  logEvent: (eventName: string, eventProperties: Js.Json.t) => ();
+  setUserProperties: (_userId: string, _userProperties: Js.Json.t) => ();
+  identify: (_userId: string) => ();
+  unidentify: () => ();
+  addToGroup: (groupType: string,
+    groupId: string,
+    groupProperties: Js.Json.t
+  ) => ();
+  logEventWithGroup: (
+    eventName: string,
+    eventProperties: Js.Json.t,
+    groupType: string,
+    groupId: string
```

Comment:
Oooh, I think named arguments would be very nice here

Reply:
```suggestion
  logEventWithGroup: (
    ~eventName: string,
    ~eventProperties: Js.Json.t,
    ~groupType: string,
    ~groupId: string
```

Reply:
or even just for the group arguments

Reply:
right, for the backward compatibility

Reply:
ah, it's a new method, so should be fine either way